cmd/deploy.go

```diff
@@ -21,29 +21,52 @@ var deployCmd = &cobra.Command{ //nolint:gochecknoglobals
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		projectId := flags.GetProjectId()
+		if projectId == "" {
+			logger.Fatal("Must provide a project ID in the --project flag")
+
+			return
+		}
+
 		path := args[0]
 		workingDir := utils.GetWorkingDir()
+		if workingDir == "" {
+			logger.Fatal("Unable to get working directory")
+
+			return
+		}
+
 		folderName := filepath.ToSlash(filepath.Join(workingDir, path))
 
 		zipPath, err := files.Zip(folderName)
 		defer files.Remove(zipPath)
 
 		if err != nil {
 			logger.FatalErr("Unable to zip folder", err)
+
+			return
 		}
 
 		gcsURL, err := storage.Upload(zipPath)
 		if err != nil {
 			logger.FatalErr("Unable to upload to Google Cloud Storage", err)
+
+			return
 		}
 		logger.Debugf("Uploaded to %v", gcsURL)
 
 		apiKey := viper.GetString("key")
+		if apiKey == "" {
+			logger.Fatal("Must provide an API key in the --key flag")
```

Comment:
There's an env var now too

Reply:
This still works for the env var. 
<img width="970" alt="Screenshot 2023-10-15 at 8 45 28 PM" src="https://github.com/amp-labs/cli/assets/18614743/3c876522-3030-47c9-97d0-f27d388b0d3e">